modules/diffusion/dfem_diffusion_solver.cc

```diff
@@ -302,7 +302,7 @@ DFEMDiffusionSolver::Execute()
     const auto cc_nodes = cell_mapping.GetNodeLocations();
     const auto fe_vol_data = cell_mapping.MakeVolumetricFiniteElementData();
 
-    const auto imat = cell.material_id;
+    const auto iblk = cell.block_id;
```

Comment:
just noting this occurrence, but could we use something else for a name like `block_id` or `iblock`. i just find that `iblk`, even though as short as `imat` could be obscure but i leave this as a comment, not a requested change. @wdhawkins  opinion?

Reply:
I think I prefer `block_id = cell.block_id`.

Reply:
This should be done